### PR TITLE
new cask for Roon Bridge

### PR DESCRIPTION
**Pull request by GeraldLoeffler**
**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

_In the following questions `<cask>` is the token of the cask you're submitting._

After making all changes to a cask, verify:

- [ x] The submission is for [a stable version](https://docs.brew.sh/Acceptable-Casks#stable-versions) or [documented exception](https://docs.brew.sh/Acceptable-Casks#but-there-is-no-stable-version).
- [ x] `brew audit --cask <cask>` is error-free.
- [ x] `brew style --fix <cask>` reports no offenses.

Additionally, **if adding a new cask**:

- [x ] Named the cask according to the [token reference](https://docs.brew.sh/Cask-Cookbook#token-reference).
- [x ] Checked the cask was not [already refused](https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues).
- [ x] Checked the cask is submitted to [the correct repo](https://docs.brew.sh/Acceptable-Casks#finding-a-home-for-your-cask).
- [ x] `brew audit --new-cask <cask>` worked successfully.
- [ x] `brew install --cask <cask>` worked successfully.
- [ x] `brew uninstall --cask <cask>` worked successfully.

### CODE REVIEW

File: Casks/roon-bridge.rb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+cask "roon-bridge" do
+  version "1.0"
+  sha256 :no_check
+
+  url "https://download.roonlabs.com/builds/RoonBridge.dmg"
+  name "Roon Bridge"
+  desc "Music player network extender"
+  homepage "https://roonlabs.com/"
+
+  app "RoonBridge.app"
```
Comment (neutric) [inline Casks/roon-bridge.rb:10]: Thank you for your contribution to `homebrew-cask` @GeraldLoeffler!

Please try to add a [`zap` stanza](https://docs.brew.sh/Cask-Cookbook#stanza-zap). The linked documentation provides some recommended methods to create it. Bonus points if you also do so for the `roon` cask. ;)

Comment (spinofdoom) [reply Casks/roon-bridge.rb]: I'm not the original author of the commit, but I downloaded `RoonBridge` from the `url` source and found the `zap` entry to be `zap trash: "~/Library/RoonBridge"`.

Comment (GeraldLoeffler) [reply Casks/roon-bridge.rb]: thank you @spinofdoom , i'll add that zap entry

**Comment by GeraldLoeffler** [comment]
can we re-open this please so that i can add the `zap trash: "~/Library/RoonBridge"` stanza kindly suggested by @spinofdoom ? Thanks

**Comment by neutric** [comment]
@GeraldLoeffler PR is reopened, you are good to go!

**Comment by GeraldLoeffler** [comment]
thanks @neutric , done from my side

**Review by spinofdoom — COMMENTED** [review]
Switch the single quotes (‘ ‘) to double quotes (“ “) on line 17, and delete any empty spaces whatsoever on line 16 while still keeping it a separate line (i.e. the whitespace error) and it should be good to go.

**Comment by GeraldLoeffler** [comment]
done - btw, https://docs.brew.sh/Cask-Cookbook#summary-of-keys says `trash: (string or array) - single-quoted, absolute paths of files or directory trees to move to Trash.` which is why i've used single quotes originally

**Comment by neutric** [comment]
Thanks for pointing that out, this should be fixed. Based on your latest commit msg I won't bother asking if you'd like to take that on. :D

**Review by SMillerDev — APPROVED** [review]

**Comment by bevanjkay** [comment]
Thank you @GeraldLoeffler

**Pull request merged**